lib/remotemanager/README.md

```diff
@@ -0,0 +1,11 @@
+remotemanager Library
+=====================
+
+MicroPython library to manage data streams and data points of a Digi Remote
+Manager account through HTTP protocol.
+
+Supported platforms
+-------------------
+
+* Digi XBee3 Cellular LTE-M/NB-IoT - minimum firmware version: 11410
```

Comment:
Like I called out in the top-level README, we support XBee Cellular LTE Cat 1 Verizon and 3G Global...

Reply:
Done.